java.lang.RuntimeException: An error occured while executing doInBackground()
       at android.os.AsyncTask$3.done(AsyncTask.java:300)
       at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
       at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
       at java.util.concurrent.FutureTask.run(FutureTask.java:242)
       at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
       at java.lang.Thread.run(Thread.java:841)
Caused by: java.lang.IllegalArgumentException: HTTP entity may not be null
       at org.apache.http.util.EntityUtils.toString(EntityUtils.java:110)
       at org.apache.http.util.EntityUtils.toString(EntityUtils.java:146)
       at eu.nullbyte.android.urllib.Urllib.open(Urllib.java:150)
       at eu.nullbyte.android.urllib.Urllib.open(Urllib.java:147)
       at eu.nullbyte.android.urllib.Urllib.open(Urllib.java:139)
       at com.liato.bankdroid.banking.banks.nordea.Nordea.preLogin(Nordea.java:197)
       at com.liato.bankdroid.banking.banks.nordea.Nordea.login(Nordea.java:239)
       at com.liato.bankdroid.banking.banks.nordea.Nordea.update(Nordea.java:264)
       at com.liato.bankdroid.DataRetrieverTask.doInBackground(DataRetrieverTask.java:89)
       at com.liato.bankdroid.DataRetrieverTask.doInBackground(DataRetrieverTask.java:40)
       at android.os.AsyncTask$2.call(AsyncTask.java:288)
       at java.util.concurrent.FutureTask.run(FutureTask.java:237)
       at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
       at java.lang.Thread.run(Thread.java:841)